lib/game/player.go

```diff
@@ -231,25 +297,11 @@ func (p *Player) simulateBlaster(controls ControlState) *coords.World {
 		return nil
 	}
 
-	ray := physics.NewRay(p.pos, p.look)
+	ray := physics.NewRay(p.Body().Pos, p.Body().Dir)
 	// We let the user shoot in the past, but they always move in the present.
-	hitPos, hitEntity := p.world.FindFirstIntersect(p, controls.ViewTimestamp, ray)
-	if hitEntity != nil {
-		p.world.DamageEntity(p.name, 40, hitEntity)
+	hitPos, hitBiotic := p.world.FindFirstIntersect(p, controls.ViewTimestamp, ray)
+	if hitBiotic != nil {
+		p.world.DamageBiotic(p.name, 40, hitBiotic)
```

Comment:
Right now we only have Biotics. So it doesn't seem to make sense for this change. Unless we want to be able to do something like shooting the world items. When we add damageable objects like doors, then this makes more sense. I want to get this PR in so I can fix the protocol.